backup_and_restore/replacing-failed-master.adoc

```diff
@@ -18,10 +18,15 @@ To replace a single master host:
 
 * xref:../backup_and_restore/replacing-failed-master.adoc#restore-remove-failed-master_replacing-failed-master-host[Remove the member from the etcd cluster].
 
+* If the etcd certificates for the master host have expired, xref:../backup_and_restore/replacing-failed-master.adoc#restore-regenerate-etcd-certs_replacing-failed-master-host[regenerate etcd certificates].
+
 * xref:../backup_and_restore/replacing-failed-master.adoc#restore-add-master_replacing-failed-master-host[Add the member back to the etcd cluster].
 
 // Removing a failed master host from the cluster
 include::modules/restore-remove-failed-master.adoc[leveloffset=+1]
 
+// Regenerating etcd certificates
```

Comment:
Switch this one with L31 and L32. Change the title as: 
Regenerating etcd certificates and recovering/adding the member to the cluster

Reply:
@retroflexer Would it make sense to use "generate" instead of "regenerate" throughout this, since there's a chance it could be being done for the first time?

Reply:
I agree.